tool/tsh/common/tsh.go

```diff
@@ -579,8 +580,12 @@ type CLIConf struct {
 	// lookPathOverride overrides return of LookPath(). used in tests.
 	lookPathOverride string
 
-	// HardwareKeyAgent determines whether the daemon will run the hardware key agent.
-	HardwareKeyAgent bool
+	// HardwareKeyAgentServer determines whether `tsh daemon`` will run the hardware key agent server.
+	HardwareKeyAgentServer bool
+	// disableHardwareKeyAgentClient determines whether the client will attempt to connect
+	// to the hardware key agent. Some commands, like login, are better with the
+	// direct PIV service.
```

Comment:
Could you briefly mention why they're better with the direct PIV service?

Reply:
Added "so that prompts are not split between processes".